package.json

```diff
@@ -94,8 +95,6 @@
     "babel-plugin-inline-replace-variables": "^1.3.1",
     "babel-plugin-lodash": "^3.3.4",
     "babel-plugin-rewire-ts": "1.4.0",
-    "better-docs": "2.7.3",
```

Comment:
doc generation wasn't built into our CI process and from what I can tell, this was unused with typedoc